list_to_email_converter.py

```diff
@@ -0,0 +1,215 @@
+import os
+from urllib.parse import quote
+import sys
+import argparse
+
+user_input = input("Write the path of your file or folder: ")
```

Comment:
Use argparse to get user input. Otherwise, shell auto-completion will not work for example.